### PR TITLE
BZ2051610: start VM in paused state

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.9, 4.10, 4.11

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2051610

Link to docs preview: https://deploy-preview-44846--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-using-the-cli-tools.html#virt-virtctl-commands_virt-using-the-cli-tools

Additional information:

### CODE REVIEW

**Comment by fabiand** [comment]
lgtm!

File: modules/virt-virtctl-commands.adoc [hunk 1]
```diff
@@ -38,7 +38,7 @@ The following table contains the `virtctl` commands used throughout the {VirtPro
 |Command |Description
 
 |`virtctl start <vm_name>`
-|Start a virtual machine.
+|Start a virtual machine. Optionally, start the virtual machine in a paused state by running `virtctl start --paused <vm_name>`. This enables you to interrupt the boot process from the VNC console.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply modules/virt-virtctl-commands.adoc:41]: Another thought. This is an important feature and was requested by customers. We might want to put it on its own row, after virtctl start. I realize that "--paused" is an option but it's an important option, something to consider.

**Pull request merged**